src/components/popover/tab/keypress/keypress-tab.component.ts

```diff
@@ -66,6 +66,15 @@ export class KeypressTabComponent implements OnChanges, Tab {
         return keystrokeAction.scancode > 0 || keystrokeAction.modifierMask > 0;
     }
 
+    onKeysCapture(event: {code: number, left: boolean[], right: boolean[]}) {
+        if (event.code) {
```

Comment:
Is it intentional that you don't clear the scancode when the event's code doesn' exists?